docs/developer/app-store/apps/emails-and-messages/overview.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Overview
+sidebar_position: 1
+---
+
+# Introduction
+
+Emails and messages (EAM) application is responsible for sending emails and messages to the customers. It is connected to the Saleor API via webhooks, which notify the application about the events. The messages are delivered by external providers configured in the application.
+
+## Suported providers
+
+Emails and Messages app currently supports the following providers:
+
+- [Sendgrid](sendgrid) - Create templates in the Sendgrid visual editor and send them to your customers.
+- [SMTP](smtp) - Use your own SMTP server to send emails. Email templates are created using MJML and handlebars.
+
+## Supported events
+
+### Account confirmation
+
+A message containing an account activation token after registration. To activate the account, your storefront should perform an API call according to [the documentation](../../../users#registration-with-email-confirmation).
+
+If you don't want to use the email confirmation, you can disable it in the Saleor dashboard:
+
+1. Open the Saleor dashboard
+1. Navigate to the Configuration page
+1. Click on the Site settings card
+1. Update the `User registration` form
+
+The event has been implemented using the `NOTIFY` webhook.
```

Comment:
**suggestion:** AFAIK, the webhook name is `NOTIFY_USER`.

**suggestion (non-blocking)**: maybe we can link to the [webhook type enum](https://docs.saleor.io/docs/3.x/api-storefront/webhooks/enums/webhook-sample-event-type-enum#code-style-fontweight-normal-webhooksampleeventtypeenumbnotify_userbcode)?

```suggestion
The event has been implemented using the `NOTIFY_USER` webhook.
```

Reply:
Good catch!